variables.tf

```diff
@@ -70,3 +70,8 @@ variable "labels" {
   description = "The resource labels to represent user provided metadata."
   default     = {}
 }
+
+variable "enable_apis" {
```

Comment:
Do we have any other modules that provide this opt-in management of associated APIs? If so, what variable naming do they use? If not, shall we standardize on this variable for enabling APIs?

Reply:
I did some searching yesterday and didn't find any modules that do this. Something related is the project-factory's [`activate_apis` variable](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/cb6c82a18ead581033c0cadfca0842b5c06b9356/variables.tf#L75-L79), but that's quite distinct. It's my understanding this will be the first use of what is to become a standard.